plugins/module_utils/entity.py

```diff
@@ -411,9 +411,12 @@ def _fetch_url(
             return {"status_code": status_code}
 
         if resp_json is None:
-            if len(info.get("msg")) > 0:
-                resp_json_msg = "{0}".format(info.get("msg"))
-            else:
+            try:
+                if len(info.get("msg")) > 2:
```

Comment:
Thats fine, let it be 0 so that we can back trace any error message in our product code base.

Reply:
done

Reply:
@konstruktoid What happens if msg key is not present in info ? I am just thinking it can case TypeError ? As len(None) can cause that. Does ansible's fetch_url confirms that msg will be always a string ?

Reply:
Since you are only checking ValueError. Exception can be raised here.

Reply:
yeah, perhaps i should just simplify it like https://github.com/nutanix/nutanix.ansible/pull/486/files#diff-be0a07cfc89126d4e28a8fbd2d2716635f9ab383eb08f54b504429dd647853efL399-L400

Reply:
@konstruktoid there is a small syntax issue. Please check latest comment